README.md

```diff
@@ -6,10 +6,6 @@
     <img src='https://github.com/qBraid/pyqasm/actions/workflows/main.yml/badge.svg' alt='CI'>
 </a>
 
```

Comment:
Use status badge in this format to be consistent the the ones below:

```
[![CI](https://github.com/qBraid/pyqasm/actions/workflows/main.yml/badge.svg?branch=main)](https://github.com/qBraid/pyqasm/actions/workflows/main.yml)
```

I only used HTML syntax for the pyqasm logo so that I could use right align